tests/BCM56850.pl

```diff
@@ -716,5 +723,6 @@ sub test_no_lag_label
 test_brcm_full_to_empty_no_queue_no_ipg_no_buffer_profile;
 test_brcm_query_attr_enum_values_capability;
 test_brcm_query_object_type_get_availability;
+test_macsec_p2p_establishment
```

Comment:
if you are adding new test, you can add it on the top of the list, then if it will fail,, you will not need to wait for all previous tests to be executed, you will save a lot of time on this